sdk/include/psdk/winbase.h

```diff
@@ -1151,7 +1151,11 @@ typedef struct _SYSTEM_POWER_STATUS {
 	BYTE ACLineStatus;
 	BYTE BatteryFlag;
 	BYTE BatteryLifePercent;
+#if _WIN32_WINNT < _WIN32_WINNT_WIN10
```

Comment:
Do we use `Reserved1` anywhere? If not, we don't need this ifdef. It's unconditionally called `SystemStatusFlag` in PSDK.

Reply:
ReactOS itself does not use this field.
Nonetheless, shouldn't PSDK and test be both consistent with S2003 target? As in usable separately, especially such a compile-time test.

Reply:
No, our headers should match the latest official ones.

Reply:
Ah. Done.